Services.Test/UserManagementClientTest.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.IoTSolutions.UIConfig.Services.Diagnostics;
+using Microsoft.Azure.IoTSolutions.UIConfig.Services.Exceptions;
+using Microsoft.Azure.IoTSolutions.UIConfig.Services.External;
+using Microsoft.Azure.IoTSolutions.UIConfig.Services.Http;
+using Microsoft.Azure.IoTSolutions.UIConfig.Services.Runtime;
+using Moq;
+using Newtonsoft.Json;
+using Services.Test.helpers;
+using Xunit;
+
+namespace Services.Test
+{
+    public class UserManagementClientTest
+    {
+        private const string MOCK_SERVICE_URI = @"http://mockauth";
+
+        private readonly Mock<IHttpClient> mockHttpClient;
+        private readonly UserManagementClient client;
+        private readonly Random rand;
+
+        public UserManagementClientTest()
+        {
+            this.mockHttpClient = new Mock<IHttpClient>();
+            this.client = new UserManagementClient(
+                this.mockHttpClient.Object,
+                new ServicesConfig
+                {
+                    UserManagementApiUrl = MOCK_SERVICE_URI
+                },
+                new Logger("UnitTest", LogLevel.Debug));
+            this.rand = new Random();
+        }
+
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
+        public async Task GetAllowedActions_ReturnValues()
+        {
+            var userObjectId = this.rand.NextString();
+            var roles = new List<string> { "Admin" };
```

Comment:
Don't we have constants for these values?

Reply:
no, this service shouldn't actually know the roles, as it just pulls them from the authorization token and then queries against the auth service. This is only used for testing